src/gui/owncloudgui.cpp

```diff
@@ -552,6 +558,11 @@ void ownCloudGui::setupContextMenu()
     }
 }
 
+void ownCloudGui::setupContextMenuIfVisible()
+{
+    if (_contextMenu && _contextMenu->isVisible())
```

Comment:
For platform tray icons, `_contextMenu->isVisible()` will always return `false`.

Though I am not saying that this change is wrong. I am going to cherry-pick it to the Ubuntu packaging anyway, because without it the context menu is updated every 30 seconds, and Qt [quickly runs out](https://bugs.launchpad.net/ubuntu/+source/unity/+bug/1635577/comments/31) of items IDs.